src/com/company/Main/Main.java

```diff
@@ -0,0 +1,294 @@
+package com.company.Main;
+
+import java.util.*;
+
+public class Main {
+
+    enum Colours {
+
+        WHITE,
+        BLUE,
+        BLACK,
+        RED,
+        YELLOW,
+        MOON,
+        ORANGE
+
+    }
+    private enum FuelTank{
+
+        FUEL_TANK_100l(100),
+        FUEL_TANK_150l(150),
+        FUEL_TANK_200l(200);
+
+        private int value;
+
+        FuelTank(int value){
+
+            this.value = value;
+
+        }
+
+    }
+
+    protected static class MyClass {
```

Comment:
I don't understand the name "My Class". Why it Is protected?
Each class move to separate file and create package if needed